tests/ICU4N.Tests/Dev/Test/Util/Trie2Test.cs

```diff
@@ -764,13 +764,24 @@ private Trie2Writable genTrieFromSetRanges(int[][] ranges)
         private void checkTrieRanges(String testName, String serializedName, bool withClone,
                 int[][] setRanges, int[][] checkRanges)
         {
-            string ns = typeof(Trie2Test).GetTypeInfo().Namespace;
+            string ns =
+#if FEATURE_TYPEEXTENSIONS_GETTYPEINFO
+            typeof(Trie2Test).GetTypeInfo().Namespace;
```

Comment:
Please indent this 4 spaces so it is clear it is a line continuation, not a new line.

Reply:
Okay, i will fix this in the next commit.